packages/common/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@atproto/common",
   "version": "0.1.1",
-  "main": "src/index.ts",
+  "main": "dist/index.js",
```

Comment:
```suggestion
  "main": "src/index.ts",
```

Reply:
you'll be happy to know i battled with this for 30 min yesterday wondering why my changes to common were not affecting anything 🙃